From fc72753a9d940639b5b2094811f47f3f62b6be33 Mon Sep 17 00:00:00 2001
From: devkinetic <devkinetic@114708.no-reply.drupal.org>
Date: Wed, 27 Jan 2021 17:02:42 -0500
Subject: [PATCH] Issue #3195017: Support SVG Image Field

---
 src/SiteSettingsLoader.php | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/SiteSettingsLoader.php b/src/SiteSettingsLoader.php
index 559a4ca..f5efbf0 100755
--- a/src/SiteSettingsLoader.php
+++ b/src/SiteSettingsLoader.php
@@ -195,6 +195,7 @@ class SiteSettingsLoader {

             case 'image':
             case 'file':
+            case 'svg_image_field':
               $build[$key] = $this->addSupplementaryImageData($build[$key], $field);
               break;
           }
--
GitLab
